packages/cms/pages/resources/resource/resource.tsx

```diff
@@ -220,22 +236,10 @@ export const Table: React.FunctionComponent = () => {
     ]
   }, [resource])
 
-  const items = [
-    {
-      id: '1',
-      firstName: 'John',
-      lastName: 'Doe',
-      github: 'johndoe',
-      dateOfBirth: Date.now(),
-      nationality: 'NL',
-      online: true
-    }
-  ]
-
   const pagination: EuiBasicTableProps<any>['pagination'] = {
-    pageIndex: 0,
-    pageSize: 10,
-    totalItemCount: 344,
+    pageIndex,
+    pageSize,
+    totalItemCount: items.length,
```

Comment:
This is actually supposed to be the total count from the metadata returned by the API